guides/source/active_record_callbacks.md

```diff
@@ -431,7 +431,7 @@ end
 NOTE: The `:on` option specifies when a callback will be fired. If you
 don't supply the `:on` option the callback will fire for every action.
 
-Since using `after_commit` callback only on create, update, or delete is
+Since using `after_commit` callbacks only on create, update, or delete are
```

Comment:
The language in this section seems to prefer "the `after_xxxx` callback", i.e. referring to the trigger rather than the targets.  So for consistency, what do you think about:

```suggestion
Since using the `after_commit` callback only on create, update, or delete is
```

Reply:
You're right; looks good to me :)